lib/mote_sms/message.rb

```diff
@@ -95,8 +101,20 @@ def to(*numbers)
     #
     # Returns result of transport#deliver.
     def deliver(options = {})
+      Kernel.warn 'Message#deliver is deprecated and will be removed from MoteSMS. Please use #deliver_now'
+      deliver_now options
+    end
+
+    def deliver_now(options = {})
+      Kernel.warn 'options[:transport] in Message#deliver_now is deprecated and will be removed from MoteSMS' if options[:transport]
       transport = options.delete(:transport) || self.transport || MoteSMS.transport
       transport.deliver(self, options)
     end
+
+    def deliver_later(options = {})
+      return Kernel.warn 'options[:transport] is not supported in Message#deliveer_later' if options.delete(:transport)
+      raise 'huhuhu' unless defined?(ActiveJob)
+      DeliveryJob.set(options).perform_later @from.to_s, @to.normalized_numbers, @body, options
```

Comment:
is the `raise` required, isn't DeliveryJob failing automatically if ActiveJob is not available?